tests/bluechi_test/fixtures.py

```diff
@@ -45,6 +53,79 @@ def bluechi_ctrl_svc_port() -> str:
     return _get_env_value('BLUECHI_CTRL_SVC_PORT', '8420')
 
 
+@pytest.fixture(scope='session')
+def machines_ssh_user() -> str:
+    """Returns the user for connecting to the available hosts via SSH"""
+
+    return _get_env_value('SSH_USER', 'root')
+
+
+@pytest.fixture(scope='session')
+def machines_ssh_password() -> str:
+    """Returns the password for connecting to the available hosts via SSH"""
+
+    return _get_env_value('SSH_PASSWORD', 'root')
```

Comment:
Wouldn't it be better to use `None` as the default value for password?

Reply:
Since `_get_env_value` returns only a string (based on the typing system), I'd return just an empty string.